nengo_gui/main.py

```diff
@@ -10,9 +10,12 @@
 from nengo_gui.password import gensalt, hashpw, prompt_pw
 
 
+logger = logging.getLogger(__name__)
+
+
 def old_main():
-    print("'nengo_gui' has been renamed to 'nengo'.")
-    print("Please run 'nengo' in the future to avoid this message!\n")
```

Comment:
This should always be visible to the user and not end up in some log file. Thus, it should stay a `print` or even better go to `sys.stderr`.